yt_dlp/extractor/twitch.py

```diff
@@ -383,10 +437,41 @@ def _extract_info_gql(self, info, item_id):
             'was_live': True,
         }
 
+    def _extract_storyboard(self, item_id, storyboard_json_url, duration):
+        spec = list(self._download_json(storyboard_json_url, item_id, "Downloading storyboard metadata JSON") or [])
```

Comment:
1. why cast to list?
2. Use single quotes
3. Must be non-fatal
```suggestion
        spec = self._download_json(storyboard_json_url, item_id, 'Downloading storyboard metadata JSON', fatal=False) or []
```